web/frontend/package-lock.json

```diff
@@ -1,242 +1,8 @@
 {
   "name": "trento",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This indicates you're using an old version of `npm`. Could you please update to v7 (`npm update -g npm` I think), reset the lock file, and re-run npm install?

Reply:
Done in https://github.com/trento-project/trento/pull/24/commits/fd4fae72b1666dece96dcecc48e9a0b94f75f121